### ISSUE TITLE
Is that possible to support kibana 5.1.* also?

### ISSUE THREAD

**Issue by ssoto** [ommsolutions/kibana_ext_metrics_vis#5]
Following the installation process the Visualization icon isn't shown.

Thanks!

**Comment by gregwjacobs** [comment]
+1

**Comment by stefandoorn** [comment]
@gregwjacobs I'm experiencing the same. Did you find any solution in the meanwhile?

**Comment by stefandoorn** [comment]
Btw, what I noticed when unpacking I'm getting the following structure:

`/usr/share/kibana/plugins/kibana/extended_metric_vis`

I just removed the kibana folder in between, so it becomes:

`/usr/share/kibana/plugins/extended_metric_vis`

Restarted Kibana and now it seems to work. My log files were not writeable, that was preventing me from starting it again apparently.

**Comment by stefandoorn** [comment]
Ok. I didn't do `npm install`. Just killed Kibana couple of times and out of a sudden it started to work.. I'm on 5.1.1. (I adjusted that in package.json).

**Comment by gregwjacobs** [comment]
I tried just dropping the contents into ```/usr/share/kibana/plugins/extended_metric_vis``` and a restart (several times) ... same result after also setting version to ```5.1.1.``` to make my install version.  I'd assume ```npm install``` reads any dependencies defined in package.json and installs them, otherwise does nothing else when its local hard copy inside the plugins folder. There are no external requirements defined in this plugins ```packages.json``` anyways. 

Assuming further debug required with some extended logging turned on in Kibana perhaps to debug?

**Comment by gregwjacobs** [comment]
I see the issue. So basically by default run kibana directly to consume its default STDOUT logging to debug, or configure it in its settings .yml -  default will not have logs saved anywhere when started by init.d/service/systemd whatever. 

Adding the plugin seems to trigger the system to re-generate assets, so adding the new plugin may require some patience while it spins up.  As you suggested I just had to copy it in place, modify the ```packages.json```, restart it and 'wait' for it to finish. Took about 1-2 minutes.  If it had other dependaces ,using ```npm install``` in the same folder would be required when doing a 'manual' install. I assume the kibana installer helper does this otherwise when It can pull down the asset via HTTP(s). 

Example logs
```
yourname@yourhost:/usr/share/kibana/bin$ sudo ./kibana
  log   [14:46:29.046] [info][optimize] Optimizing and caching bundles for monitoring, kibana, timelion and status_page. This may take a few minutes
```

It had to wait and let it work a few minutes hard and then it worked fine. 

Basically it appears a new 'release' zip just has to be made on this github site, as version 5.1.1 and kibana users should be able to install directly without issues. Confirmed it works on my end. 

![image](https://cloud.githubusercontent.com/assets/1749292/22068749/00a89286-dd64-11e6-96fa-599b14d241c7.png)

**Comment by gregwjacobs** [comment]
Issued a pull request via #6 with confirmed fix / adjustment to work with 5.1.1

**Comment by patil-akshay** [comment]
as we know this plug_in works for 5.0.0 only to work on 5.1.1 follow below steps

Just extract the file modify the package.json 
{
  "name": "extended_metric_vis",
  "version": "0.1.0",
  "main": "index.js",
  "kibana": {
    "version": "5.1.1"
  },
  "author": "Olaf Horstmann <oh@omm-solutions.de>",
  "website": "http://www.omm-solutions.de"
}

like tihs an enjoy

**Comment by githubAmin2** [comment]
my kibana is 5.4.0, but when i manual unzip plugin package under /usr/share/kibana/plugins and  modify the package.json
{
"name": "extended_metric_vis",
"version": "0.1.0",
"main": "index.js",
"kibana": {
"version": "5.4.0"
},
"author": "Olaf Horstmann oh@omm-solutions.de",
"website": "http://www.omm-solutions.de"
}

The icon from visualize is not shown after many times restart kibana 5.4

**Comment by TechWatching** [comment]
Same for me, I also have the log when I start Kibana : 
[09:44:55.743] [warning][plugins] Skipping non-plugin directory at D:\Kibana\kibana-5.4.0-windows-x86\plugins\extended_metric_vis
